apps/nrm/views/partials/does-pv-have-children-yes-amount.html

```diff
@@ -0,0 +1,3 @@
+<fieldset id="does-pv-have-children-yes-fieldset" class="panel-indent">
+    {{#renderField}}does-pv-have-children-yes-amount{{/renderField}}
```

Comment:
as above?

will this not just generate if it's in `index.js` i.e. https://github.com/UKHomeOffice/modern-slavery/blob/master/apps/nrm/index.js

is it because it's a partial?

Reply:
Yes, it was loading the field separately as well as within the radio button child element. So I did the HTML file as an override. Is there a better way? @sulthan-ahmed

Reply:
I think the way you did it was the best